dbt-snowflake/src/dbt/adapters/snowflake/relation_configs/dynamic_table.py

```diff
@@ -112,7 +124,10 @@ def parse_relation_results(cls, relation_results: RelationResults) -> Dict[str,
             "query": dynamic_table.get("text"),
             "target_lag": dynamic_table.get("target_lag"),
             "snowflake_warehouse": dynamic_table.get("warehouse"),
+            "catalog": SnowflakeCatalogConfig.parse_relation_results(relation_results),
```

Comment:
I think this got removed in the recent catalog implementation.

Reply:
yea got some weird merge conflicts, will clean up